src/pages/index.js

```diff
@@ -14,28 +15,80 @@ import {
   addNewCardForm,
   validationSettings,
   profileEditForm,
+  selectors,
+  avatarEditButton,
+  avatarEditForm,
 } from "../utils/constants.js";
 import "../pages/index.css";
 
-//Card Section Instance
-const section = new Section(
-  {
-    items: initialCards,
-    renderer: (cardData) => {
-      const cardElement = renderCard(cardData);
-      section.addItem(cardElement);
-    },
+const cardSelector = selectors.cardTemplate;
+
+//Api
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "67d17471-1da7-4346-ba00-92490de8ca42",
+    "Content-Type": "application/json",
   },
-  cardList
-);
+});
+
+Promise.all([api.getUserInfo(), api.getInitialCards()])
+  .then(([userData, cardData]) => {
+    userInfo.setUserInfo(userData);
+    userInfo.setAvatar(userData.avatar);
+    userId = userData._id;
+    newCardSection = new Section(
+      {
+        items: cardData,
+        renderer: (data) => {
+          const newCard = renderCard(data);
+          newCardSection.addItem(newCard);
+        },
+      },
+      selectors.cardSection
+    );
+    newCardSection.renderItems();
+  })
+  .catch(console.error);
+
+//Card Section Instance
+// Promise.all([api.getInitialCards(), api.getUserInfo()])
+//   .then(([initialCards, userData]) => {
+//     userInfo.setUserInfo(userData.name, userData.about);
+//     userInfo.setUserAvatar(userData.avatar);
+//     section = new Section(
+//       {
+//         items: initialCards,
+//         renderer: (cardData) => {
+//           const cardElement = renderCard(cardData);
+//           section.addItem(cardElement);
```

Comment:
**Could be improved**

Please, remove **commented code** to clean up the project